region/maridia/inner-pink/West Sand Hole.json

```diff
@@ -360,6 +360,58 @@
         "A diagonal ShineSpark will most likely not have any horizontal movement when performed from the sand."
       ]
     },
+    {
+      "link": [1, 7],
+      "name": "Jump Assist, Use Flash Suit",
+      "requires": [
+        "canPlayInSand",
+        {"or": [
+          "HiJump",
+          "canSpringBallJumpMidAir"
+        ]},
+        {"useFlashSuit": {}},
+        {"or": [
+          {"shinespark": {"frames": 10, "excessFrames": 3}},
+          {"and": [
+            "HiJump",
+            {"shinespark": {"frames": 7, "excessFrames": 3}}
+          ]}
+        ]}
+      ],
+      "flashSuitChecked": true,
+      "devNote": "FIXME: Is it possible to get onto the ledge with a sandfall bounce?"
+    },
+    {
+      "link": [1, 7],
+      "name": "Break Block on Descent, Use Flash Suit",
+      "requires": [
+        "canPlayInSand",
+        {"or": [
+          "h_canUsePowerBombs",
+          "Wave"
+        ]},
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 18, "excessFrames": 3}}
+      ],
+      "flashSuitChecked": true,
+      "note": "Break the block on the descent with Wave or a Power Bomb, jump off the sand, then spark diagonally out of the water."
+    },
+    {
+      "link": [1, 7],
+      "name": "Hero Shot Break Block, Use Flash Suit",
+      "requires": [
+        "canPlayInSand",
+        "canHeroShot",
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 27, "excessFrames": 5}}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "From the bottom of the sand, shoot upwards to break the block then Shinespark up. To use a flash suit in deep sand:",
```

Comment:
You can jump from the sand after shooting to save some health.  Does make it easier to fail, or just sink into the sand.